manual/de/Help_ch_Account-Actions.xml

```diff
@@ -1025,25 +1025,6 @@
           <listitem>
             <para>Ein optionaler Nummernschlüssel wie in <xref linkend="accts-code" /> beschrieben.
             </para>
-<!-- FIXME source: new note -->
-            <note>
-              <para>Ähnlich der Kontonummer bietet der Typ eines Wertpapiers oder einer Währung eine Möglichkeit,
-                seine Investments sachlich zu sortieren. Grundsätzlich bietet es sich an, sie nach
-                Risiko zu gruppieren. Zum Beispiel kann als erste Ebene der Typ des Wertpapiers
-                angegeben werden und als optionale 2. Ebene die Nominalwährung.
-<screen>:
-|
-+--Bond
-|   |--CHF
-|   |--EUR
-+--Aktie
-|   |--CHF
-|   |--EUR
-|   |--GBP
-|
-:</screen>
```

Comment:
niedrige Prio: statt `:` `&vellip;` oder so

Reply:
@fellen 
`[WARN] FOUserAgent - Glyph "⋮" (0x22ee, ellipsisvertical) not available in font "Courier"`.

Das wird scheinbar von dem Zeichensatz nicht unterstützt.

Reply:
Kein Problem, ist nur Kosmetik.